data/dataset/postgres_example_test_dataset.yml

```diff
@@ -127,6 +127,19 @@ dataset:
             data_categories: [system.operations]
 
       - name: payment_card
+        # a sample partitioning for functional testing - not realistic!
+        fides_meta:
+          partitioning:
+            field: billing_address_id
+            windows:
+              - start: "0"
```

Comment:
might be nice to allow proper ints here, but we do also need to support timestamps, and easiest for now is just to accept generic strings...